### PR TITLE
Make sure composed change is compatible with IE. 

### PR DESCRIPTION
**Pull request by craigpetchell**
Also added test-watch script to make local testing with IE easier.

This fixes the IE failures introduced by #767

### CODE REVIEW

**Comment by craigpetchell** [comment]
Also for reference, local run of tests, since browserstack didn't run as this PR is from a fork:

```
IE 10.0.0 (Windows 7 0.0.0): Executed 221 of 221 SUCCESS (1.428 secs / 0.928 secs)
Firefox 48.0.0 (Mac OS X 10.11.0): Executed 222 of 222 SUCCESS (1.88 secs / 1.165 secs)
IE 11.0.0 (Windows 7 0.0.0): Executed 222 of 222 SUCCESS (1.621 secs / 0.938 secs)
Chrome 53.0.2785 (Mac OS X 10.11.6): Executed 223 of 223 SUCCESS (1.762 secs / 1.187 secs)
TOTAL: 888 SUCCESS
```

File: package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
   },
   "scripts": {
     "test": "sk-tests",
+    "test-watch": "sk-tests-watch",
```
Comment (treshugart) [inline package.json]: Nit: are really only for automated npm stuff; the `sk-*` commands are public API from `skatejs-build`.

**Comment by treshugart** [comment]
lgtm

File: src/api/emit.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-      new Event(); // eslint-disable-line no-new
+      new TheEvent('emit-init'); // eslint-disable-line no-new
     } catch (e) {
       return undefined;
     }
   }
-  return Event;
-})(window.CustomEvent);
+  return TheEvent;
+})(window.Event);
```
Comment (jpnelson) [inline src/api/emit.js]: is this polyfilled? What was the problem?

Comment (craigpetchell) [inline src/api/emit.js]: No it's not. And it's used further down instead of CustomEvent (See pr #767 for the other half of the change).

**Comment by jpnelson** [comment]
lgtm

**Comment by treshugart** [comment]
lgtm

**Comment by jpnelson** [comment]
lgtm

**Pull request merged**